components/PlantDetails.js

```diff
@@ -33,6 +33,41 @@ const Figure = styled.figure`
   margin: 1rem;
 `;
 
+const StyledPeriodSummaryContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 1rem;
+`;
+
+const StyledPeriodSummary = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
Here also the flex-direction is in default column you dont need to write it here :)